src/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -1,22 +1,79 @@
+using Esfa.Recruit.Employer.Web.Configuration;
 using Esfa.Recruit.Employer.Web.Configuration.Routes;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Esfa.Recruit.Employer.Web.Extensions;
 
 namespace Esfa.Recruit.Employer.Web.Controllers
 {
+    [Route("accounts/{employerAccountId}/dashboard")]
     public class DashboardController : Controller
     {
         ILogger<DashboardController> _logger;
+        private readonly IOptions<AuthenticationConfiguration> _authConfig;
+        private readonly IOptions<ExternalLinksConfiguration> _extLinksConfig;
 
-        public DashboardController(ILogger<DashboardController> logger)
+        public DashboardController(ILogger<DashboardController> logger, IOptions<AuthenticationConfiguration> authConfig, IOptions<ExternalLinksConfiguration> extLinksConfig)
         {
             _logger = logger;
+            _authConfig = authConfig;
+            _extLinksConfig = extLinksConfig;
         }
 
-        [HttpGet, Route("accounts/{employerAccountId}/dashboard", Name = RouteNames.Dashboard_Index_Get)]
+        [HttpGet("", Name = RouteNames.Dashboard_Index_Get)]
         public IActionResult Index()
         {
             return View();
         }
+
+        [HttpGet("change-password", Name = RouteNames.Dashboard_ChangePassword)]
+        public IActionResult ChangePassword(string returnUrl)
+        {
+            var url = string.Format(_extLinksConfig.Value.ManageApprenticeshipSiteChangePasswordUrl, _authConfig.Value.ClientId, $"{Request.GetRequestUrl()}{returnUrl}");
```

Comment:
I think the returnUrl should be Url encoded as its going to be a parameter value